### DISCUSSION TITLE
great theme but.....

### DISCUSSION THREAD

**Discussion by fonik000** [lobehub/sd-webui-lobe-theme#498] [General]
This expansion is really a dream of appearance, but I often have problems with performance:
- "after start" Lora's loading infinite (always gray)
- The operating speed is very bad ( its lagy?)

What could be the reason for this?
on legacy Theme all runs fast.

Is that the same with you?

**Comment by canisminor1990** [comment]
try adding `--no-gradio-queue` to the COMMANDLINE_ARGS
